src/sentry/integrations/gitlab/client.py

```diff
@@ -0,0 +1,69 @@
+from __future__ import absolute_import
+
+from six.moves.urllib.parse import quote
+
+from sentry.integrations.client import ApiClient, OAuth2RefreshMixin
+from sentry.integrations.exceptions import ApiError
+
+
+class GitLabApiClient(ApiClient, OAuth2RefreshMixin):
+    api_version = '/v4/'
+
+    def __init__(self, base_url, access_token):
```

Comment:
I think we might have to pass the integration here, or at least the `integration.metadata['verify_ssl']` for all the request. In GHE we have it hard coded to `False` but that will change too.

Reply:
Hopefully I'll get the Install flow out soon (hopefully merged tomorrow) so we can test this